src/containers/RepositoriesContainer/RepositoriesContainer.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+// Actions
+import * as actionCreators from './RepositoriesDucks.js';
```

Comment:
No, but only because the only reducer I export is the default export (Which * skips)

That said, I'm not entirely sold on this approach yet either. It's the recommendation from the Ducks approach, and it's very neat in controlled examples, but I'm nervous it will complicate testing (Unless all my unit tests go through the main reducer itself, rather than exposing the reducers it is composed of)

I've defaulted with it for now to see how well it works. Little uncomfortable, but maybe that's good? :)